follower_default_send_mail_false/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "follower_default_send_mail_false",
```

Comment:
```suggestion
    "name": "Mail Send Mail Default False",
```
Let's change the module name to `mail_send_mail_default_false`.